File: BooWho/Models/House.cs
namespace BooWho.Models
{
    public class House
    {
        public int Id { get; set; }
        public int UserProfileId { get; set; }
        public string Address { get; set; }
        public string ImageUrl { get; set; }
        public string Notes { get; set; }
        public UserProfile UserProfile { get; set; }
        public UserType UserType { get; set; }
        public GhostType GhostType { get; set; }

    }
}

File: BooWho/client/src/Components/House/House.js
import React from "react";

const House = ({ house }) => {
  return (
    <div className="text-left px-2">
      {house.imageUrl}

      <p>
        <strong>{house.address}</strong>
      </p>
      <p>{house.notes}</p>
    </div>
  );
};

export default House;
